app/controllers/users_controller.rb

```diff
@@ -10,6 +11,10 @@ def edit
   def password
   end
 
+  def show
+    redirect_to user_path if @other_user.id == current_user.id
```

Comment:
do we really need this here? It should be fine showing user from this page as well